docs/getting_started.rst

```diff
@@ -62,6 +62,13 @@ Installation Quick Reference Guide
       You may get a slightly different value but if your NengoFPGA system
       is functioning correctly, this should be near 0.001.
 
+   .. tip::
+      If you run the ``basic_example.py`` script without any errors
```

Comment:
If the ``basic_example.py`` script runs without any errors... ?

I'm trying to word it in such a way where it implies the script starts running without producing any errors, but doesn't imply that the script actually finishes running without errors. Not sure how to reword it that way... haha

Reply:
Yea I retyped that a few times as well, I will think about it over lunch.

Reply:
Pushed a revision